src/renderer/index.css

```
@@ -1,9 +1,15 @@
 @import 'tailwindcss';
+@import '@asgardex/asgardex-theme/theme.css';
 
 /* Class-based dark mode (matches v3 darkMode: 'class') */
 @custom-variant dark (&:where(.dark, .dark *));
 
 @theme {
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)